cups/translate-attrs.go

```diff
@@ -67,6 +67,16 @@ func getUUID(printerTags map[string][]string) string {
 }
 
 func getState(printerTags map[string][]string) cdd.CloudDeviceStateType {
+	// offline-report reason is set when printer is offline (e.g. USB cable is disconnected)
```

Comment:
Two things:
- You should probably use `getVendorState(printerTags)` which properly removes the `-report` and other suffixes.
- Does the `printer-state` tag change in some other way? See https://tools.ietf.org/html/rfc2911#section-4.4.11. Or any other tag? Maybe `printer-is-accepting-jobs` ? (Which we need anyway for #233.)

I will run a diff (with ipptool) on my printer.